site/recipes/recipe_adaptable_polling_source.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Changing a Polled Source Stream's Period
+---
+
+The [Writing a Source Function](recipe_source_function.html) recipe introduced the basics of creating a source stream by polling a data source periodically.
+
+Oftentimes, a user wants the poll frequency to be adaptable rather than static.  For example, an event such a sudden rise in a temperature sensor may motivate more frequent polling of the sensor and analysis of the data until the condition subsides.  A change in the poll frequency may be driven by locally performed analytics or via a command from an external source.
```

Comment:
Missing an "as" between "such" and "a" in: _For example, an event such a sudden rise..._